app/views/home/feed.xml.builder

```diff
@@ -0,0 +1,16 @@
+atom_feed do |feed|
+  feed.title t('feed.title')
+  feed.updated @projects[0].updated_at unless @projects.empty?
+
+  @projects.each do |project|
+    feed.entry(project) do |entry|
+      entry.title "#{project.chapter.name} – #{project.title}"
+      entry.content(project.about_project, type: 'html')
```

Comment:
This should be `project.funded_description` (the editable text during funding), not `about_project` (which is the original application text).